mlflow/protos/databricks_artifacts.proto

```diff
@@ -79,6 +79,28 @@ service DatabricksMlflowArtifactsService {
       visibility: PUBLIC_UNDOCUMENTED,
     };
   }
+
+  rpc getCredentialsForTraceDataDownload (GetCredentialsForTraceDataDownload) returns (GetCredentialsForTraceDataDownload.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "GET",
+        path: "/mlflow/traces/{trace_id}/credentials-for-data-download"
+        since {major: 2, minor: 0},
+      }],
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc getCredentialsForTraceDataUpload (GetCredentialsForTraceDataUpload) returns (GetCredentialsForTraceDataUpload.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "GET",
+        path: "/mlflow/traces/{trace_id}/credentials-for-data-upload"
+        since {major: 2, minor: 0},
+      }],
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
```

Comment:
Can we rename these in accordance with the backend renaming?

Reply:
updated!